permissions/Contacts.permission

```diff
@@ -19,6 +19,9 @@ dbus-user.talk org.sailfishos.Messages
 # To launch contacts import wizard
 dbus-user.talk com.jolla.contacts.ui
 
+dbus-user.own org.nemomobile.contacts.sqlite.*
+dbus-user.broadcast org.nemomobile.contacts.sqlite.*=org.nemomobile.contacts.sqlite.*@/org/nemomobile/contacts/sqlite
```

Comment:
Just wondering would it allow skipping the service registration in app side if this was *=org.nemomobile...

Reply:
I didn't check it again but IIRC no, it doesn't work like that. Currently it's just not possible to allow signals from unique bus names.

Reply:
FWIW such a format is already used on Base.permission for standard interfaces.

Reply:
Funny. I think it can be used as allow from any well-known bus name but unique bus names are still not allowed in the code.

Reply:
Anyhow, could think *=... being more correct here as it should be possible to trigger the signal regardless of the registered services.

Reply:
Updated to use *=...
Verified that the app side still requires the service registration with this change.